aptos-move/framework/aptos-framework/sources/aptos_account.spec.move

```diff
@@ -131,10 +131,10 @@ spec aptos_framework::aptos_account {
         let coin_store_source = global<coin::CoinStore<AptosCoin>>(account_addr_source);
         let balance_source = coin_store_source.coin.value;
 
-        requires forall i in 0..len(recipients):
-            recipients[i] != account_addr_source;
-        requires exists i in 0..len(recipients):
-            amounts[i] > 0;
+        // requires forall i in 0..len(recipients):
```

Comment:
I cannot remember why we need to have this. Since `pragma verify = false`, I just commented them out.

Reply:
It seems that these lines are unnecessary, and we can remove them. We can't require them, and they are not even aborting conditions.